recipes/bokeh/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.12.0" %}
```

Comment:
Is it alright if we start with `0.11.1`? I need that release for other reasons. We can bump the version to `0.12.0` in the feedstock.